src/api/rif-marketplace-cache/rns/common.ts

```diff
@@ -24,17 +24,16 @@ export type RnsAPIService = Modify<
   }
 >
 
-export const isSupportedToken = (
+export const isSupportedRNSToken = (
   token: string,
-): boolean => SUPPORTED_TOKENS[token]
+): boolean => rnsSupportedTokens.some((t: string) => t === token)
 
-const { contractAddresses } = network
-contractAddresses.rbtc = ZERO_ADDRESS
+//  - Supported Token Address-to-token records
+export const rnsNftAddrTokenRecord: Record<string, BaseToken> = allNftAddresses
```

Comment:
```suggestion
export const rnsTokenAddressRecord: Record<string, BaseToken> = allNftAddresses
```

I think `allNftAddresses` should also be renamed in its file